src/command_modules/azure-cli-network/HISTORY.rst

```diff
@@ -2,8 +2,14 @@
 
 Release History
 ===============
+
+unreleased
+++++++++++++++++++
+* `nic create/update`: Add support for `--accelerated-networking`.
+* `nic create`: Remove non-functional `--internal-dns-name-suffix` argument.
```

Comment:
This should be marked as `BC`, though i am fine we can release w/o deprecating process